grawlix-cms/_admin/book.page-edit.php

```diff
@@ -692,11 +692,76 @@
 	$headline_output = $page-> pageInfo['blog_title'];
 }
 
+//Script for adding HTML tags, just as in the Create Page file.
 $blog_output = <<<EOL
 <label for="blog_headline">Headline</label>
 <input type="text" name="blog_headline" id="blog_headline" value="$headline_output"/>
 <label for="blog_post">Post</label>
-<textarea name="blog_post" id="blog_post" rows="7">{$page-> pageInfo['blog_post']}</textarea>
+<script language="javascript" type="text/javascript">
+
+function addText(Text,Message)
+{
+  var obj = document.form.blog_post;
+
+  obj.focus();
+
+  if (document.selection && document.selection.createRange)  // Internet Explorer
+  {
+sel = document.selection.createRange();
+if (sel.parentElement() == obj)  sel.text = Text;
+  }
+
+  else if (typeof(obj) != "undefined")  // Firefox
+  {
+var longueur = parseInt(obj.value.length);
+var selStart = obj.selectionStart;
+var selEnd = obj.selectionEnd;
+
+obj.value = obj.value.substring(0,selStart) + Text + obj.value.substring(selEnd,longueur);
+  }
+
+  else obj.value += Text;
+
+  obj.focus();
+}
+
+
+function addTags(Tag,fTag,Message)
+{
+  var obj = document.form.blog_post;
+
+  obj.focus();
+
+  if (document.selection && document.selection.createRange)  // Internet Explorer
+  {
+sel = document.selection.createRange();
+if (sel.parentElement() == obj)  sel.text = Tag + sel.text + fTag;
+  }
+
+  else if (typeof(obj) != "undefined")  // Firefox
+  {
+var longueur = parseInt(obj.value.length);
```

Comment:
This file also has "longueur" where an English and more descriptive name would be more appropriate.